distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -29,6 +29,7 @@ start.solr=true
 # DDF Environment Settings
 ddf.etc=${karaf.etc}
 ddf.data=${karaf.data}
+ddf.ingest=${karaf.home}/ingest
```

Comment:
❓ What happens if an admin changes this to live outside the distribution? Won't this not work and the policy have to be updated? Should we add a note here? To me I'd almost say this should not be configurable - since they can specify a path in the command (which will also require a policy change).

Reply:
I was able to ingest files from ~/Download successfully, outside of the distribution.

Reply:
I made it configurable because I didn't want it to look like I was doing a &#189; 🐴 job.  ☺
@lessarderic Do you want the default ingest directory to be configurable in system properties for fixed to `DDF_Home/ingest/`

Reply:
`~/Download` will work because it is under the user's home directory. Anywhere else it would fail.

Reply:
Yeah, by default DDF has read/write permissions to anything in the user's home directory.
https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/security/default.policy#L440-L441

Reply:
I wouldn't make it configurable for now. Like @peterhuffer mentioned, they can always specify a path when they run the command if they want to ingest from somewhere else.